inspirehep/modules/workflows/workflows/hep_ingestion.py

```diff
@@ -125,6 +130,11 @@ class mysource(HEPIngestion):
             with_author_keywords=True,
         ),
         filter_core_keywords,
+        guess_categories,
+        IF(is_experimental_paper, [
+            guess_experiments,
+        ]),
+        guess_keywords,
```

Comment:
@jalavik I see (as it should indeed be) that arXiv harvesting and hep ingestion workflows are quite similar. Is there a way to factor out the common part, so that enhancement such as the ones in this PR can be written only once?